src/server/scripts/Northrend/Naxxramas/instance_naxxramas.cpp

```diff
@@ -182,6 +182,12 @@ class instance_naxxramas : public InstanceMapScript
                     case NPC_SIR:
                         SirGUID = creature->GetGUID();
                         break;
+                    case NPC_GLUTH:
+                        GluthGUID = creature->GetGUID();
+                        break;
+                    case NPC_ZOMBIE_CHOW:
+                        ZombieChowGUIDs.push_back(creature->GetGUID());
```

Comment:
Why do you rely on InstanceScript in order to do something that should be handled by the BossAI. Do not clutter instance events like this, please.